### PR TITLE
fpga: update demo

### PR DESCRIPTION
**Pull request by bart0sh**
Updated FPGA demo screencasts:
- used latest kustomize setup
- got rid of cloning github repository
- renamed test-fpga-region.yaml -> demo/test-fpga-orchestrated.yaml
- updated readme

### CODE REVIEW

File: demo/readme.md [hunk 1]
```diff
@@ -90,21 +90,20 @@ The demos begin with a fully [configured Kubernetes cluster](https://kubernetes.
 ### Demo steps
 
 1. Validate the status of the [Kubernetes cluster](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/).
-2. Clone the [Intel Device Plugins for Kubernetes source](https://github.com/intel/intel-device-plugins-for-kubernetes).
```
Comment (mythi) [inline demo/readme.md:93]: is this a miss?

Comment (mythi) [reply demo/readme.md:93]: OK, I see this is indeed desired but we may want to fix 1...3, 4, 5

Comment (bart0sh) [reply demo/readme.md:93]: @mythi thank you for the review. updated the PR.

File: demo/screencast-fpga-preprogrammed.sh [hunk 2]
```diff
@@ -71,50 +70,39 @@ screen1()
...
 {
   clear
-  out '4. Run OPAE workload that uses NLB3 bitstream'
+  out '3. Run OPAE workload that uses NLB3 bitstream'
   out 'Check if devices are programmed with NLB3:'
   command 'cat /sys/class/*/*/*/afu_id'
   out 'Run workload:'
-  command 'cat plugins/demo/test-fpga-preprogrammed.yaml'
-  command 'kubectl create -f plugins/demo/test-fpga-preprogrammed.yaml'
+  command 'curl https://raw.githubusercontent.com/intel/intel-device-plugins-for-kubernetes/master/demo/test-fpga-preprogrammed.yaml' 100
```
Comment (mythi) [inline demo/screencast-fpga-preprogrammed.sh:97]: it looks curl is redundant if you deploy from the repo directly??

Comment (bart0sh) [reply demo/screencast-fpga-preprogrammed.sh:97]: yes, I just wanted to show the content of that .yaml

Comment (mythi) [reply demo/screencast-fpga-preprogrammed.sh:97]: ah, ok!

**Pull request merged**